_rules/aria-required-owned-element-bc4a75.md

```diff
@@ -65,6 +57,12 @@ If the [explicit semantic role][] on the target element is incorrectly used, and
 - [Required Owned Element](https://www.w3.org/TR/wai-aria-1.1/#mustContain)
 - [Owned Element](https://www.w3.org/TR/wai-aria-1.1/#dfn-owned-element)
 
+HTML elements with an [implicit semantic role][] corresponding with [explicit semantic role][] are not tested in this rule because some of these elements have additional requirements. For example, an element that has an implicit semantic role that is identical to its explicit semantic role is a `ul` element that has `role="list"`. However, there is no native HTML element to "group" `li` elements in a `ul` element.
+
+The unresolved issues with how [Digital Publishing WAI-ARIA Module][] (DPUB ARIA 1.1) uses role inheritance to define the [required owned element][] is limiting the scope of this rule to [WAI-ARIA 1.1][] roles only. It is because of the deviations between DPUB ARIA 1.1 and [WAI-ARIA 1.1][] models. The [WAI-ARIA Graphics Module][] does not include any [required owned element][].
```

Comment:
```suggestion
The unresolved issues with how [Digital Publishing WAI-ARIA Module][] (DPUB ARIA 1.1) uses role inheritance to define the [required owned element][] limit the scope of this rule to [WAI-ARIA 1.1][] roles only. The [WAI-ARIA Graphics Module][] does not include any [required owned element][].
```

Reply:
I didn't mean to change the aria-required-own-element in this PR. I messed up a biut by creating a branch from #1365 but I will add those changes to it. Thanks!